src/App.js

```diff
@@ -1,10 +1,12 @@
 import './App.css';
 import {Route, Routes} from "react-router-dom";
 import {Home} from "./pages/Home";
-import {Navbar} from "./components/common/Navbar";
+// import {Navbar} from "./components/common/Navbar";
+
 import {Games} from "./pages/Games";
 import {Activities} from "./pages/Activities";
 import {activities, games} from "./data/content";
+import Navbar from './components/common/NewNavbar';
```

Comment:
I don't think you were permitted to remove the old navbar.

Reply:
Done